### ISSUE TITLE
npm run start:prod error when using boilerplate with semantic-ui, semantic-ui-react, and redux-form

### ISSUE THREAD

**Issue by karloespiritu** [mxstbr/react-boilerplate#1318]
The boilerplate (v3.3.3) won't load on `npm run start:prod` when using it with `redux-form` and `semantic-ui`. In the browser console, this error appears:

```
 Uncaught TypeError: Cannot read property 'call' of undefined(…)  in http://localhost:3000/main.07993bcd469b17e4dedf.js
```

<img width="846" alt="console-error-log" src="https://cloud.githubusercontent.com/assets/219083/21008527/134d3ace-bd7d-11e6-90e8-d5200f21573c.png">

This does not occur when just using `npm run start` but only when using `npm run start:prod`. I suspect this has something to do with how `webpack` builds the dependencies.

I recreated the issue by just adding `semantic-ui` and `semantic-ui-react` to a fork of [https://github.com/carlos-peru/react-boilerplate-redux-form](https://github.com/carlos-peru/react-boilerplate-redux-form), a project that demonstrates integrating `redux-form` to `react-boilerplate`.

My repo: [https://github.com/karloespiritu/react-boilerplate-redux-form-semantic](https://github.com/karloespiritu/react-boilerplate-redux-form-semantic) demonstrates this issue. 

I would assume someone else might have encountered this issue since those 3 packages -- `redux-form`, `semantic-ui`, `semantic-ui-react` -- are pretty common to include in react projects.

Any help/clue/direction to resolve this issue is highly appreciated. :) Thanks.

**Comment by karloespiritu** [comment]
I found a fix to my issue, and it is caused by webpack DedupePlugin()  in `internals/webpack/webpack.prod.babel.js`. To fix the error, I just removed the line:

```
// Merge all duplicate modules		
new webpack.optimize.DedupePlugin(),
```
This plugin should be removed immediately from the latest version of this project as it causes errors that ONLY occurs in the production build, that is -- only when loading the application on `npm run start:prod`. 

It appears this issue is very common because as soon as you add new packages to this boilerplate the same issue occurs in production build, as described in the following issues:

- [Issue with production build](https://github.com/mxstbr/react-boilerplate/issues/1268)
- [Issue with adding material-ui library](https://github.com/mxstbr/react-boilerplate/issues/1210)

It is very difficult to resolve an issue that only occurs in production build since all the files are already minified.  Even worse --  `npm install`, `npm run build`, or `npm run start` does not report the error, ONLY when you load the production build (minified version) in your browser.

I see that this this plugin is already removed in the `mxstbr:dev` branch, but I hope this will be merged in `master` branch as soon as possible so other users of this boilerplate won't have to spend hours of trying to solve the issue when it is just a matter of removing one line.

**Comment by Augustin82** [comment]
Thank you @karloespiritu for saving us hours of work!

**Comment by mxstbr** [comment]
Yes, we'll be releasing 3.4 shortly! Thanks for digging into this @karloespiritu, sorry for the troubles!

**Comment by mxstbr** [comment]
@gihrig @Dattaya can we hotfix this to `master`? Could you submit and merge a PR?

**Comment by JakeLane** [comment]
I noticed that the PR (#1324) was rejected as it was going to be shipped with 3.4. It's been 13 days since this issue was solved and 3.4 still hasn't been released. This is a single line that needs to be changed and it is breaking building production in master.

I heavily urge reconsideration of #1324.
